.circleci/config.yml

```diff
@@ -329,10 +329,15 @@ workflows:
             - prep-deps
             - prep-build
             - prep-build-mv2
-            - trigger-beta-build
             - prep-build-mmi
             - prep-build-flask
             - prep-build-flask-mv2
+            - prep-build-test
```

Comment:
These are all required by the prepublish job because it uses the builds persisted to the workspace by these jobs.

Some of these were _indirectly_ guaranteed to be present in the workspace due to them being required by some other listed dependency here. But best not to rely on that.